app/templates/userProfile.html

```diff
@@ -1,13 +1,78 @@
 {% extends 'base.html' %}
 
 {% block content %}
-{% if current_user.is_authenticated and (current_user.role == 'admin' or current_user.user_id == user.user_id) %}
-<a href="{{ url_for('user.update', user_id=user.user_id) }}" type="button" class="btn btn-info">Update Profile</a>
-{% endif %}
 
-{% for attribute, value in user.__dict__.items() %}
-<br /> {{ attribute }}::{{ value }}
-{% endfor %}
+  <div class="container py-5 h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col col-lg-6 mb-4 mb-lg-0">
+        <div class="card mb-3" style="border-radius: .5rem;">
+          <div class="row g-0">
+            <div class="col-md-4 gradient-custom text-center text-white" style="border-top-left-radius: .5rem; border-bottom-left-radius: .5rem;">
+              <img
+                src="https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-chat/ava1-bg.webp"
+                alt="Avatar"
+                class="img-fluid my-5"
+                style="width: 80px;"
+              />
+              <h5>{{ current_user.username }}</h5>
```

Comment:
replace `current_user` to `user`.

Reply:
sorry, I forgot to push. the change has been done. 
![image](https://user-images.githubusercontent.com/79505566/147740952-98d04b16-f9c9-4d40-b798-bfb0c36fe757.png)